quickevent/app/plugins/Oris/src/orisimporter.cpp

```diff
@@ -748,6 +749,11 @@ void OrisImporter::importRegistrations()
 {
 	int sport_id = eventPlugin()->eventConfig()->sportId();
 	int year = QDate::currentDate().addMonths(-2).year();
+
+	bool ok;
+	year = QInputDialog::getInt(nullptr, tr("Import ORIS Registrations"), tr("Year of registration:"), year, year-2, year+2, 1, &ok);
```

Comment:
Jako max číslo by asi dávalo smysl povolovat jen year+1, minimální bych viděl asi na year-1

Reply:
Ja o tom tak hluboce nepremyslel. Ze o jeden bylo jasne. Tak jsem dal dva "pro jistotu". Nicemu to nevadi, a kdybych nejake specialni pouziti zapomnel, tak to pokryje. Vic nez dva asi nikdo nevyuzije.